napari/conftest.py

```diff
@@ -608,6 +608,9 @@ def single_shot(msec, reciver, method=None):
                 t.timeout.connect(reciver)
             else:
                 t.timeout.connect(getattr(reciver, method))
+            calling_place = _get_calling_place(2)
+            if "superqt" in calling_place and "throttler" in calling_place:
+                calling_place += f" - {_get_calling_place(3)}"
```

Comment:
What is the relation of this change to this PR?

Reply:
Try to debug the failing test (not terminated throttler). Unfortunately, since I added this change, the test is not failing with this error message....